src/components/InteractiveStepSubHeader.tsx

```diff
@@ -11,26 +9,26 @@ import * as Expensicons from './Icon/Expensicons';
 import PressableWithFeedback from './Pressable/PressableWithFeedback';
 import Text from './Text';
 
-const propTypes = {
+type InteractiveStepSubHeaderProps = {
     /** List of the Route Name to navigate when the step is selected */
-    stepNames: PropTypes.arrayOf(PropTypes.string).isRequired,
+    stepNames: readonly string[];
```

Comment:
shouldn't be an array of kinfd of union?

Reply:
why?